art.taunoerik.tauno-serial-plotter.yml

```diff
@@ -0,0 +1,37 @@
+app-id: art.taunoerik.tauno-serial-plotter
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: tauno-serial-plotter.py
+
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=network
+  - --device=all
+
+modules:
+  - python3-requirements.json
+
+  - name: tauno-serial-plotter
+    buildsystem: simple
+    build-commands:
+      - install -D src/tauno-serial-plotter.py /app/bin/tauno-serial-plotter.py
+      - install -Dm644 art.taunoerik.tauno-serial-plotter.desktop /app/share/applications/Tauno-serial-plotter.desktop
+      - install -Dm644 src/icons/tauno-plotter.svg /app/bin/icons/tauno-plotter.svg
+      - install -Dm644 src/icons/tauno-plotter.svg ${FLATAPK_DEST}/share/icons/hicolor/scalable/apps/tauno-plotter.svg
+      - install -Dm644 src/icons/arrow_down.svg /app/bin/icons/arrow_down.svg
+      - install -Dm644 src/icons/minus.svg /app/bin/icons/minus.svg
+      - install -Dm644 src/icons/plus.svg /app/bin/icons/plus.svg
+    sources:
+      - type: archive
+        url: https://github.com/taunoe/tauno-serial-plotter/archive/refs/tags/v1.18.tar.gz
+        sha256: 39b5741e85d78a354ed8e875675cb5dfe0f2e05e01f7df264d9ca98edff9e3e3
+      - type: file
+        path: art.taunoerik.tauno-serial-plotter.appdata.xml
```

Comment:
like the .deestkop file this should be submitted to be maintained upstream for a future release